scripts/adi_env.tcl

```diff
@@ -16,6 +16,7 @@ if [info exists ::env(ADI_GHDL_DIR)] {
 
 # Define the supported tool version
 set required_vivado_version "2023.1"
+
```

Comment:
I think is good the way it is now; anyway, this change is not related to the topic of the PR

Reply:
Could you revert the change, as it is not related to the topic?